lib/open_meteo/response_wrapper.rb

```diff
@@ -0,0 +1,41 @@
+module OpenMeteo
+  ##
+  # Wrap the JSON body response from the OpenMeteo request.
+  #
+  class ResponseWrapper
+    def initialize(config: OpenMeteo::Client::Config.new, entity: OpenMeteo::Entities::Forecast)
```

Comment:
I would probably not set a default, but move this to caller, even thought there is only one caller for now. It would make the RespnseWrapper spec more explicit as well.

Reply:
Ahh true, I didn't remember that we will have more different entities. I will move this as a parameter of the wrap method, like we always do.